narwhals/_duckdb/utils.py

```diff
@@ -228,14 +233,14 @@ def _non_nested_native_to_narwhals_dtype(duckdb_dtype_id: str, version: Version)
     dtypes.Int16: duckdb_dtypes.SMALLINT,
     dtypes.Int32: duckdb_dtypes.INTEGER,
     dtypes.Int64: duckdb_dtypes.BIGINT,
-    dtypes.Int128: DuckDBPyType("INT128"),
+    dtypes.Int128: duckdb_dtypes.DuckDBPyType("INT128"),
```

Comment:
Should we use

```suggestion
    dtypes.Int128: duckdb_dtypes.HUGEINT,
```

(and `dtypes.UInt128: duckdb_dtypes.UHUGEINT,` below)?